lib/gcloud/storage/bucket/list.rb

```diff
@@ -51,6 +51,67 @@ def next
         end
 
         ##
+        # Retrieves all buckets by repeatedly loading {#next} until {#next?}
+        # returns `false`. Calls the given block once for each bucket, which is
+        # passed as the parameter.
+        #
+        # An Enumerator is returned if no block is given.
+        #
+        # This method may make several API calls until all buckets are
+        # retrieved. Be sure to use as narrow a search criteria as possible.
+        # Please use with caution.
```

Comment:
Just now saw this comment. The missing `@param` and `@yieldparam` docs in the latest commits.